guides/common/modules/proc_configuring-a-physical-interface.adoc

```diff
@@ -0,0 +1,37 @@
+[id="configuring-a-physical-interface"]
+= Configuring a physical interface
+
+You can configure a physical interface for a host.
+
+.Procedure
+
+. Navigate to the *Add Interface* form:
+.. In the {ProjectWebUI}, navigate to *Hosts* > *All Hosts*.
+.. Click *Edit* next to the host you want to edit.
+.. On the *Interfaces* tab, click *Add Interface*.
+
+. Specify the general interface settings:
+.. Specify a *MAC address*.
+This setting is required.
+.. Specify the *Device Identifier*, for example `eth0`.
+
+. Configure network and DNS settings:
+.. Specify the *DNS name* associated with the host's IP address.
+.. Select a domain from the *Domain* list.
+.. Select a subnet in the *IPv4 Subnet* or *IPv6 Subnet* list.
+.. Specify the *IPv4 address* or *IPv6 address*.
++
+[NOTE]
+====
+Managed interfaces with an assigned DHCP {SmartProxyServer} require this setting for creating a DHCP lease.
+DHCP-enabled managed interfaces are automatically provided with a suggested IP address.
+====
+
+. Specify interface management options:
```

Comment:
This kind of subprocedure, where each step starts with "Select whether ..." is not easy for a user to scan.